source/clear-linux/guides/maintenance/fix-broken-install.rst

```diff
@@ -0,0 +1,79 @@
+.. _fix-broken-install:
+
+Fix a broken installation of |CL-ATTR|
+######################################
+
+This guide explains how to fix a broken installation of |CL| using a live
+desktop image on a USB. It's assumed you already installed |CL| on a
+target system, but your operating system does not boot or function properly.
+This guide addresses installation issues that may have impacted the
```

Comment:
This process can only verify and fix files that are owned by the swupd in /usr.  Files outside of this path (i.e. /home/, /etc, /var, etc.) cannot be repaired by this process.